tests/topotests/pim_igmp_vrf/r1/pimd.conf

```diff
@@ -9,20 +9,33 @@ hostname r1
 ! debug pim zebra
 ! debug pim bsm
 !
-interface r1-eth0
+vrf blue
+exit-vrf
```

Comment:
Can you please open an issue to track this one? will try to find time to fix it.

Reply:
Can't reproduce this right now with the current version of the testcase with unified config. Will try to find an easy way how to reproduce this and then open an issue.

Reply:
OK, also happens with unified config as long as you don't load it using `router.load_frr_config()` but like before with per-daemon calls to `router.load_config()`

Reply:
[#18429](https://github.com/FRRouting/frr/issues/18429)